source/lambda/jobresultprocessor/lambda_function.py

```diff
@@ -153,7 +153,7 @@ def processRequest(request):
         if key not in comprehendAndMedicalEntities:
             comprehendAndMedicalEntities[key] = val
         else:
-            comprehendAndMedicalEntities[key].add(val)
+            comprehendAndMedicalEntities[key].append(val)
```

Comment:
We should perhaps make this change in SyncProcessor lambda as well.

Reply:
+1